src/pages/entity/[entityName]/create/[frameworkKey].page.tsx

```diff
@@ -78,34 +65,34 @@ const EntityIntroPage = () => {
         }
       });
     }
-  };
+  }, [createEntity, entityName, entityUUID, formUUID, parentName, parentUUID, router]);
 
   return (
     <BackgroundLayout>
       <ContentLayout>
         <LoadingContainer loading={!form}>
-          <WizardFormIntro
-            title={form?.title!}
-            //@ts-ignore
-            imageSrc={form?.banner?.url}
```

Comment:
I discovered this morning that this was being added via `appendFilesToResources` in PHP (I had thought it was simply missing, and the `ts-ignore` was hiding that fact). I'm going to make sure this gets added back in in my current ticket (TM-2415 - indexForms)

Reply:
Addressed here: https://github.com/wri/wri-terramatch-website/commit/deaf3e987c376e6797781712d253c2a3980e5d50